js/components/bookmarksToolbar.js

```diff
@@ -152,6 +152,9 @@ class BookmarkToolbarButton extends ImmutableComponent {
       })}
       draggable
       ref={(node) => { this.bookmarkNode = node }}
+      {
+        !this.isFolder ? title={this.props.bookmark.get('location')}
+      }
```

Comment:
If you make changes to your branch you can `git add -u` and then `git commit --amend` and then `git push -f` and the pull request will be updated.